ui/app/styles/components/actions.scss

```diff
@@ -117,14 +117,30 @@
       }
 
       .messages {
+        width: 100%;
         overflow: hidden;
 
-        code > pre {
-          height: 200px;
+        code {
           background-color: #0a0a0a;
           color: whitesmoke;
+          display: block;
+          overflow: auto;
+          height: 200px;
           border-radius: 6px;
           resize: vertical;
+          pre {
+            background-color: transparent;
+            color: unset;
+            overflow-anchor: none;
```

Comment:
Did you have a chance to test on Safari, where `overflow-anchor` doesn't seem to be supported?
<img width="501" alt="image" src="https://github.com/hashicorp/nomad/assets/775380/e3093a41-b06e-4445-b78d-04d6e77cf061">

Reply:
Yep, this won't work in Safari; observed behaviour there is same as pre-PR (treats this like any other scrollable element, not "sticky")